client/state/ui/my-sites/sidebar/reducer.js

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	MY_SITES_SIDEBAR_SITE_TOGGLE,
+	MY_SITES_SIDEBAR_DESIGN_TOGGLE,
+	MY_SITES_SIDEBAR_TOOLS_TOGGLE,
+	MY_SITES_SIDEBAR_MANAGE_TOGGLE,
+} from 'state/action-types';
+
+import { combineReducers } from 'state/utils';
+
+export function isSiteOpen( state = false, action ) {
+	switch ( action.type ) {
```

Comment:
these could all use a common toggleReducer that takes an action to respond to and toggles the state